docs/source/guide/release_notes.md

```diff
@@ -12,10 +12,94 @@ meta_description: Discover what's new and improved, and review bug fixes, in the
     <i class='ent'></i> 
     The release notes for Label Studio Community Edition is available on the <a href="https://github.com/heartexlabs/label-studio/releases"> Label Studio GitHub repository</a>.
 
+
+## Label Studio Enterprise 2.3
+
+This section highlights the new features, enhancements, and bug fixes in Label Studio Enterprise 2.3.
+
+### New features and enhancements 
+
+Label Studio Enterprise 2.3 introduces the following new features and enhancements.
+
+- This release introduces the Outliner feature with the following capabilities [DEV-1544]:
+    - Outliner panel
+    - Details panel
+    - Regions grouping/sorting
+    - Manual regions grouping (drag-and-drop)
+    - Hiding/locking regions
+    - Prediction indication on the region
+    - Region editor (Image segmentation)
+    - Relations list
+    - Panels drag-and-drop
+    - Panels snap-to-side
+    - Panels resizing
+
+- [internal] Show jobs by projects and organizations on **/django-rq** page [DEV-2898]. 
+- Add the ability to sustain the collapse state of the label across tasks [DEV-2755].
+- This release provides consistent ordering of the label groups [DEV-2755].
+- Implement new columns for comment management in the **Data Manager**: **Comment count**, **unresolved comment count**, **comment authors**, and **last comment date** [DEV-2672].
```

Comment:
I would group everything regarding comments into one group and start with `Introduce the [**Comments**](comments_annotations.html) section for annotation editor. ...`